src/main/java/com/ait/lienzo/client/core/shape/IDestroyable.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ait.lienzo.client.core.shape;
+
+/**
+ *  For stateful shape instances, in order to destroy its state.
+ */
+public interface IDestroyable {
```

Comment:
Interface formatted as Kie-Group class, not Lienzo's one. If we want to merge changes to upstream it will be an issue in the future.